### ISSUE TITLE
Dead code in AVX2 mq.c

### ISSUE THREAD

**Issue by thomwiggers** [joostrijneveld/MQDSS#1]
Is it me, or are these lines not used. Removing them doesn't make a difference.

https://github.com/joostrijneveld/MQDSS/blob/00608d7610262ff07b1834885d32bc3fd27ef5e1/avx2/mq.c#L286-L306

**Comment by joostrijneveld** [comment]
Yeah, looks like it.. Were you able to reproduce the testvectors with those lines removed?

**Comment by thomwiggers** [comment]
PQClean's `./test_testvectors.py` was happy, yeah

**Comment by thomwiggers** [comment]
The

```
     xi[3] = _mm256_add_epi16(mask_reduce&_mm256_cmpgt_epi16(_zero, xi[3]), xi[3]); 
```

lines don't compile on Windows, which is also something I ran into — Using `&`, `^` for SIMD isn't supported there.

Maybe I can combine these patches.

**Comment by thomwiggers** [comment]
This has long ago been fixed by #2
